docs/project/set-up-dev-env.md

```diff
@@ -88,339 +89,218 @@ To remove unnecessary artifacts:
     container.
 
-        root@31ed86e9ddcf:/go/src/github.com/docker/docker# go version
-        go version go1.4.2 linux/amd64
-
-    Similarly, if you do a `docker version` you find the container
-    has no `docker` binary.
-
-        root@31ed86e9ddcf:/go/src/github.com/docker/docker# docker version
-        bash: docker: command not found
+    ```bash
+    Successfully built 3d872560918e
+    docker run --rm -i --privileged -e BUILDFLAGS -e KEEPBUNDLE -e DOCKER_BUILD_GOGC -e DOCKER_BUILD_PKGS -e DOCKER_CLIENTONLY -e DOCKER_DEBUG -e DOCKER_EXPERIMENTAL -e DOCKER_GITCOMMIT -e DOCKER_GRAPHDRIVER=devicemapper -e DOCKER_INCREMENTAL_BINARY -e DOCKER_REMAP_ROOT -e DOCKER_STORAGE_OPTS -e DOCKER_USERLANDPROXY -e TESTDIRS -e TESTFLAGS -e TIMEOUT -v "home/ubuntu/repos/docker/bundles:/go/src/github.com/docker/docker/bundles" -t "docker-dev:dry-run-test" bash
+    root@f31fa223770f:/go/src/github.com/docker/docker#
+    ```
 
-    You will create one in the next steps.
+    At this point, your prompt reflects the container's BASH shell.
 
-4. From the `/go/src/github.com/docker/docker` directory make a `docker` binary
-with the `make.sh` script.
+5. List the contents of the current directory (`/go/src/github.com/docker/docker`).
 
-        root@5f8630b873fe:/go/src/github.com/docker/docker# hack/make.sh binary
+    You should see the image's source from the  `/go/src/github.com/docker/docker`
+    directory.
 
-    You only call `hack/make.sh` to build a binary _inside_ a Docker
-    development container as you are now. On your host, you'll use `make`
-    commands (more about this later).
+      ![List example](images/list_example.png)
 
-    As it makes the binary, the `make.sh` script reports the build's progress.
-    When the command completes successfully, you should see the following
-    output:
+6. Make a `docker` binary.
 
-	---> Making bundle: binary (in bundles/1.5.0-dev/binary)
-	Created binary: /go/src/github.com/docker/docker/bundles/1.5.0-dev/binary/docker-1.5.0-dev
+    ```bash
+    root@a8b2885ab900:/go/src/github.com/docker/docker# hack/make.sh binary
+    ...output snipped...
+    bundles/1.12.0-dev already exists. Removing.
 
-5. List all the contents of the `binary` directory.
+    ---> Making bundle: binary (in bundles/1.12.0-dev/binary)
+    Building: bundles/1.12.0-dev/binary/docker-1.12.0-dev
+    Created binary: bundles/1.12.0-dev/binary/docker-1.12.0-dev
+    Copying nested executables into bundles/1.12.0-dev/binary
+  ```
 
-        root@5f8630b873fe:/go/src/github.com/docker/docker#  ls bundles/1.5.0-dev/binary/
-        docker  docker-1.5.0-dev  docker-1.5.0-dev.md5  docker-1.5.0-dev.sha256
+7. Copy the binary to the container's `/usr/bin` directory.
 
-    You should see that `binary` directory, just as it sounds, contains the
-    made binaries.
+    ```bash
+    root@a8b2885ab900:/go/src/github.com/docker/docker# cp bundles/1.12.0-dev/binary/docker /usr/bin
```

Comment:
since there is containerd in 1.11 this should probably be:

`root@a8b2885ab900:/go/src/github.com/docker/docker# cp bundles/1.12.0-dev/binary/docker* /usr/bin`